### DISCUSSION TITLE
Not found redirection when an orgId doesn't exist

### DISCUSSION THREAD

**Discussion by aurrelhebert** [grafana/grafana#41796] [Platform]
Hello, 

When setting in URL an invalid orgaId, by default Grafana plot an empty page with `Not Found`: 

![Notexists](https://user-images.githubusercontent.com/2650661/142182629-a3f00bef-2fdf-4bd9-8eac-d5eb88cf3150.png)

Is it possible to be able in that case to plot a normal Grafana not found page as:

![Redirect](https://user-images.githubusercontent.com/2650661/142182777-802c18ba-0d6e-4331-a693-baf924bef0de.png)

Grafana version: 8.2.4

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
